cmd/create/ocmrole/cmd.go

```diff
@@ -0,0 +1,363 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ocmrole
+
+import (
+	"fmt"
+	"github.com/openshift/rosa/cmd/link/ocmrole"
```

Comment:
@vkareh classic review comment. This needs rearranging :)

Reply:
In `vs.code` I had `goimports` extension do that automatically on every save. Now with `goland` I'm lost...